src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -47,9 +47,18 @@
 )
 
 from ..dx_extract_utils.filter_to_payload import validate_JSON, final_payload
-from ..dx_extract_utils.germline_utils import get_genotype_types, get_genotype_only_types, add_germline_base_sql, \
-    sort_germline_variant, harmonize_germline_sql, harmonize_germline_results, get_germline_ref_payload, \
-    update_genotype_only_ref
+from ..dx_extract_utils.germline_utils import (
```

Comment:
import of `get_genotype_types` seems to be missing here, but it's being used here:

https://github.com/dnanexus/dx-toolkit/pull/1351/files#diff-9fa6c7c89cfc29f5fe6314b6c36a1dfab692ac8a3378d7e1d90e244ed233d40bR1037